Azure/sonic-buildimage
# Updating index value starts from 0-based in port_config.ini

**- What I did**
    Update index value from 1-based to 0-based

**- How I did it**
    The below files will be modified:
1. port_config.ini
2. sfputil.py
    
**- How to verify it**
    Execute command "sfputil show eeprom" to check the eeprom can be dumped if port is pluggged
    
**- Description for the changelog**
    Update index value from 1-based to 0-based

Signed-off-by: Sam Yang yang.kaiyu@gmail.com
